packages/editor-ui/src/plugins/telemetry/index.ts

```diff
@@ -86,9 +88,14 @@ class Telemetry {
 		}
 	}
 
-	track(event: string, properties?: IDataObject) {
+	track(event: string, properties?: ITelemetryTrackProperties) {
 		if (this.telemetry) {
-			this.telemetry.track(event, properties);
+			const updatedProperties = {
+				...properties,
+				version_cli: this.store && this.store.getters.versionCli,
```

Comment:
should not this be a trait instead the way we add instance id and user id

Reply:
I don't think it is right now based on a quick look at the code. https://github.com/n8n-io/n8n/blob/9864805666955061cc670a8a53d27fd55e003d87/packages/editor-ui/src/plugins/telemetry/index.ts#L81
should not it suffice as a trait? instead of adding it as a property to every request?